core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -615,81 +653,75 @@ var ExplorationEditorMainTab = function() {
     }
   };
 
+  /**
+   * Parse for Angular selectors and remove them.
+   * @param {string} [ruleDescription] - Interaction type.
+   * @param {string|string[]} [providedText] - Feedback text to replace with.
+   */
+  var _replaceAngularSelectors = function(ruleDescription, providedText) {
+    // Expected input = is equal to {{a|NonnegativeInt}} and
+    // {{b|NonnegativeInt}}.
+    var re = /{{[a-z][\|](.*?)}}/ig;
+    // Matched result = Array[{{a|NonnegativeInt}}}, {{b|NonnegativeInt}}]
+    var matchedString = ruleDescription.match(re);
+    // Replacing matched strings in ruleDescription with ...
```

Comment:
@seanlip Just wanna check it and see if this is ok with you? :+1:

Reply:
Hm, what exactly are you asking me to check?

If the functionality is unchanged from what it was before, I'm fine with it. But I would suggest taking a closer look at your docstrings/comments (see other comments).